src/main/java/step3/study/domain/RequestRacingDTO.java

```diff
@@ -0,0 +1,21 @@
+package step3.study.domain;
+
+import step3.study.util.Validator;
+
+public class RequestRacingDTO {
+    private final int carCount;
+    private final int tryCount;
+
+    public RequestRacingDTO(String carCount, String tryCount) {
+        this.carCount = Validator.numberCheck(carCount);
+        this.tryCount = Validator.numberCheck(tryCount);
+    }

```

Comment:
```suggestion
    public RequestRacingDTO(int carCount, int tryCount) {
        this.carCount = carCount;
        this.tryCount = tryCount;
    }
```

DTO 안에 Validator 로직을 넣는 것은 적절치 않은 것 같습니다.
DTO 는 로직 없이, 분리된 계층에게 정보를 전달해주기만 하는 역할을 해야하기 때문입니다.
다른 사람이 유지보수하던 중간에, 에러가 터져서 살펴보았더니, DTO 에서 Exception 을 발생시키고 있더라 라는 경우가 생기면,
당황스러울 것 같습니다.
Validator 의 로직은 DTO 를 export 하는 클래스로 옮겨주시기 바랍니다.